src/picongpu/include/simulation_defines/param/particle.param

```diff
@@ -134,13 +111,34 @@ namespace manipulators
     };
 
     /* definition of RandomEnableRadiation start*/
-    typedef FreeRngImpl< RandomEnabledRadiationFunctor, nvidia::rng::distributions::Uniform_float > RandomEnabledRadiation;
+    typedef generic::FreeRng< RandomEnabledRadiationFunctor, nvidia::rng::distributions::Uniform_float > RandomEnabledRadiation;
 
     /** changes the in-cell position of each particle of a species */
-    using RandomPosition = RandomPositionImpl<>;
+    using RandomPosition = unary::RandomPosition;
 
 } // namespace manipulators
```

Comment:
we should probably make `manipulator` and `filter` consistently singular as a namespace